packages/mintlist-cli/src/program.ts

```diff
@@ -0,0 +1,49 @@
+import { Command } from "commander";
+import { genTokenlist } from "./cmd/gen-tokenlist";
+import { genIndex } from "./cmd/gen-index";
+import { addMint } from "./cmd/add-mint";
+import { removeMint } from "./cmd/remove-mint";
+
+const program = new Command();
+
+program.name("mintlist-cli").description("CLI for managing mintlists").version("0.1.0");
+
+program
+  .command("add-mint")
+  .description("Add a mint to a mintlist")
+  .argument("<mintlist>", "Path to mintlist file")
+  .argument("<mint...>", "Mint(s) to add to mintlist")
+  .action(addMint);
+
+program
+  .command("remove-mint")
+  .alias("rm-mint")
+  .description("Remove a mint from a mintlist")
+  .argument("<mintlist>", "Path to mintlist file")
+  .argument("<mint...>", "Mint(s) to remove from mintlist")
+  .action(removeMint);
+
+program
+  .command("gen-tokenlist")
+  .description("Generate tokenlists from mintlists")
+  .argument("<mintlist...>", "Path to mintlist file(s)")
+  .requiredOption(
+    "--outDir <string>",
+    "Output directory for generated tokenlists",
+    "./src/tokenlists"
+  )
+  .option("--overrides <file>", "Path to overrides file", "./src/overrides.json")
```

Comment:
Hm, I wasn't able to replicate the error when this flag wasn't included. Do you remember what the error was?

In any case, i've removed it since it's not a required field.

Reply:
👍
The error was that src/overrides.json does not exist.
I think this is because the src/overrides.json file did not exist when I tried.